papers/views.py

```diff
@@ -328,7 +328,7 @@ def refetchResearcher(request, pk):
     from backend.tasks import fetch_everything_for_researcher
     fetch_everything_for_researcher.delay(pk=pk)
 
-    return redirect(reverse('researcher', researcher=pk))
+    return redirect(reverse('researcher', slug=researcher.slug))
```

Comment:
I think the pk should be required too - we cannot identify a researcher by its slug only. I assume that this part of the code is not tested yet.

Reply:
Indeed, I'm rebuilding the VM at the moment. We need both researcher and slug I think.

Reply:
and we need it to be wrapped in an `args={...}` keyword too (hence the error)

Reply:
anyway it would be good to confirm that with a unit test (I can add it)

Reply:
Tested in my VM and fixed. There was actually already a test, except it is only checking the case when the researcher is not logged in. Not sure how to fix this test :/ https://github.com/dissemin/dissemin/pull/620/files#diff-2e713353c3d10b8fe4b78e8759e15dc0R116